400-claims.md

```diff
@@ -50,13 +50,11 @@ From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/cscope.rb
 
 The CNAB Claims specification does not define where or how records are stored, nor how these records MAY be used by CNAB tooling. However, this specification describes how a CNAB-based system MUST emit the record to an invocation image, and provides some guidance on maintaining integrity of the system.
 
-This is done so that implementors can standardize on a way of relating a release claim (the record of a release) to release operations like `install`, `upgrade`, or `uninstall`. This, in turn, is necessary if CNAB bundles are expected to be executable by different implementations.
+This is done so that implementors can standardize on a way of relating a release claim (the record of a release) to release actions like `install`, `upgrade`, or `uninstall`. This, in turn, is necessary if CNAB bundles are expected to be executable by different implementations. If CNAB runtimes implement this common claims system, then tools can share responsibility for managing CNAB applications. For example, one tool may perform actions, while another may provide a visualization dashboard.
```

Comment:
My only question here, which admittedly isn't one of the parts amended, is the term `release` as used in _"... a release claim", "... the record of a release"_, etc.

Should we update to `installation` ?  Or are they indeed distinct terms?

Reply:
I'm going to do a follow-up PR on that. I need to do a full pass through the doc for the terms "release" and "installation" and make sure we consistently use them. An "installation" is when a package is installed. A "release" is each state change on an "installation" -- e.g. I install version 1, then upgrade to version 2: that is one installation with two releases. But there had been some other suggestions about how we might clarify that.